demos/yelp_demo/demo_setup/README.md

```diff
@@ -412,123 +436,201 @@ ks param set --env minikube serving modelPath ${GCS_TRAINING_OUTPUT_DIR_LOCAL}/e
 
 ## 5. Create a GKE cluster
 
-### Setup a CPU/GPU/TPU cluster
+Choose one of the following options for creating a cluster and installing
+Kubeflow with pipelines:
+
+* Click-to-deploy
+* CLI (kfctl)
+
+### Click-to-deploy
+
+This is the recommended path if you do *not* require access to GKE beta
+features such as TPUs and node auto-provisioning (NAP).
+
+Generate a web app Client ID and Client Secret by following the instructions
+[here](https://www.kubeflow.org/docs/started/getting-started-gke/#create-oauth-client-credentials).
+Save these as environment variables for easy access.
 
-Choose one of two ways to create a GKE cluster:
+In the browser, navigate to the
+[Click-to-deploy app](https://deploy.kubeflow.cloud/). Enter the project name,
+along with the Client ID and Client Secret previously generated. Select the
+desired ${ZONE} and latest version of Kubeflow, then click _Create Deployment_.
 
-1. gcloud command line OR
-1. Deployment Manager (no TPU support)
+In the [GCP Console](https://console.cloud.google.com/kubernetes), navigate to the
+Kubernetes Engine panel to watch the cluster creation process. This results in a
+full cluster with Kubeflow installed.
+
+### CLI (kfctl)
+
+If you require GKE beta features such as TPUs and node autoprovisioning (NAP), these
```

Comment:
Should we remove kfctl from the section heading since its not using kfctl?

Reply:
It does, but pretty far down. Added a subheader to make it a bit clearer.